modules/tasks/tasks/pom.xml

```diff
@@ -139,7 +145,7 @@
                             org.motechproject.tasks.repository;version=${project.version},
                             org.motechproject.tasks.service;version=${project.version},
                             org.motechproject.tasks.service.osgi;version=${project.version},
-                            org.motechproject.tasks.util;version=${project.version}
+                            org.motechproject.tasks.service.util;version=${project.version}
```

Comment:
it is no longer needed change for my test, but there is no package org.motechproject.tasks.util . I should revert this change?